.github/workflows/Release-candidate-containers.yml

```diff
@@ -0,0 +1,119 @@
+name: Release Candidate Container Build and Deployment Preparation
+on:
+  workflow_dispatch:
+    inputs: 
+        environment:
+          type: choice
+          required: true
+          options:
```

Comment:
To reduce button click you can add a line after line 7-- default: 'TEST'

And  after line 12 --  default: 'test2'